json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -504,6 +504,9 @@ public function update_settings() {
 					$wga = get_option( $option_name, array() );
 					$wga['code'] = $value['code']; // maintain compatibility with wp-google-analytics
 
+					// Allow newer versions of this endpoint to filter in additional fields
+					$wga = apply_filters( 'site_settings_update_wga', $wga, $value );
```

Comment:
Sorry, can you please document this filter [example here](https://github.com/Automattic/jetpack/blob/764a699120ca6795dac3d264207713516200ee7b/require-lib.php#L16-L23) so that it will be parsed for developer.jetpack.com?

Reply:
Yep - can do